cherry_picker/test_cherry_picker.py

```diff
@@ -222,6 +235,21 @@ def test_sorted_branch(os_path_exists, config, input_branches, sorted_branches):
     assert cp.sorted_branches == sorted_branches
 
 
+@mock.patch("os.path.exists")
+def test_invalid_branch_empty_string(os_path_exists, config):
+    os_path_exists.return_value = True
+    # already tested for require_version_in_branch_name=True below
+    config["require_version_in_branch_name"] = False
+    cp = CherryPicker(
+        "origin",
+        "22a594a0047d7706537ff2ac676cdc0f1dcb329c",
+        ["3.1", "2.7", "3.10", "3.6", ""],
+        config=config,
+    )
+    with pytest.raises(ValueError):
```

Comment:
Always use the `match=` regex with things like `pytest.raises()` / `warns()` / `deprecated()`. Otherwise, something else may start causing this exception and this text will still pass w/o notifying people that something's broken.